### ISSUE TITLE
Nx vs Turborepo performance values were incorrectly measured

### ISSUE THREAD

**Issue by jonahsnider** [nrwl/nx#8954]
### Documentation issue

- [ ] Reporting a typo
- [ ] Reporting a documentation bug
- [ ] Documentation improvement
- [x] Documentation feedback

### Is there a specific documentation page you are reporting?

https://nx.dev/guides/turbo-and-nx#tech-and-performance

### Additional context or description

![Turborepo vs Nx performance](https://nx.dev/documentation/shared/turbo-nx-perf.gif)

This GIF from the above documentation page seems to indicate that Nx is an order of magnitude faster than Turborepo.

This is not a fair comparison, `nx` was installed globally with `npm i -g nx` but `turbo` was not. `turbo` with npx (which introduces a fair deal of overhead to the benchmark, making Turborepo seem slower than it actually is) while `nx` was using a global install. The benchmarks should be redone to compare a global install of `turbo` with the global install of `nx`.

**Comment by FrozenPandaz** [comment]
You are right, turbo incurred some overhead from `npx`. We will work on amending the misleading documentation.

However, the benchmark is open for everyone to try here: https://github.com/vsavkin/large-monorepo.

**Comment by vsavkin** [comment]
@jonahsnider we discussed it on twitter when the benchmark was released. The actual benchmark was updated not to use `npx` right after it was released. The overhead of `npx` is negligible. For the benchmark above Nx is an order of magnitude faster with or without `npx`. 

There are other benchmarks that show an even bigger difference, but Turbo just cannot run them without a lot of custom setup, so we didn't show them.
